metadata.rb

```diff
@@ -15,4 +15,4 @@
   supports os
 end
 
-version          '1.2.0'
+version          '1.2.1'
```

Comment:
Undo this change, its already prep'ed for the next release and this isn't a bug fix